volumes/rescu/README.md

```diff
@@ -1,45 +1,347 @@
+
+### 2- Where is the raw data stored?	   	
+>Raw data is stored in Oracle and ITS Central. 
+Within ITS Central it is stored in Postgres database but we are unsure as to how much data is being stored in the system.
+### 3- How is the raw data stored?	
+> Unsure—particularly now that there are two streams (one in ITS Central and one in Oracle).
+
+### 4- Where is the data stored?	
+>Pulling data from databases into Postgres. Windows task scheduler is responsible for pulling the data from \\tssrv. 
+>Refer to [README_PIPELINE](rescu/README_PULL.md) for more information
+
+<!---### 5- How is the data stored?	
+>Unsure.   
+
+
+
+
+## Instrument/device or software-specific information needed to interpret the data:
+> Provide information on how data can be interpreted baed on collection device/instrument  
+
+
+
+## Standards information, if appropriate: 
+> N/A- to be added
+
+
+
+## Environmental/experimental conditions: 
+> N/A- to be added --->
+
+
+
+## Data Quality/ Describe any quality-assurance procedures performed on the data
+
+### 1- Are there known data gaps/incomplete data? 	
+>Yes, there are many data gaps.
+### 2- What are the gaps?	
+>Missing volumes due to detector issues. Data reports sent to D&A out of Oracle contain specific lane-level data. This may help isolate detectors that are down and help in closing gaps and validating data.
+
+### 3- How are data gaps/incomplete data addressed? 
+>Currently within D&A and for the purpose of Data Requests, requesters tell requestees that data is not available.  
+
+### 4- Who is responsible for addressing data gaps/incomplete data?
+>Not Clear but D&A can be contacted for the time being. 
+
+### 5- Are there data quality assessment processes for the data?
+>Yes, there is a pipeline check but it needs to be updated to add upstream validation process.
+
+### 6- How often are data quality assessment processes for the data undertaken? 
+>D&A process done daily. QA process counts the number of rows that have data with 7000 rows being the threshold.   
+ 
+<!---#### 6- Are external contractors or consultants responsible for checking data quality?
+> Unsure. 
+
+
+<!---## Data Validity- Is the data truly representative of the real world? 
+	
+### 1- Who is responsible for checking the validity of the data?	
+>Unsure. 
+It appears that validation is only done once data reaches D&A. Unclear is there are validation processes upstream. 
+
+### 2- Who should be involved when there are inconsistencies identified with the data?	
+>Unsure.
+### 3- For external data sources: is data received previously validated? 	
+>N/A
+### 4- For external data sources: if data received is previously validated, how is it validated?
+>N/A
+### 5- Are external contractors or consultants responsible for checking data validity? Who is the City staff contact who works for that vendor?	
+>N/A
+--->
+## Data Maintainance 
+## Maintenance - 
+- (Hardware/Software)
+  - Hardware
+  - ITS Central
+   - Oracle Data
+   
```

Comment:
I'm unsure if this needs to be here in point form - I think it is covered under heading 1 below.

Reply:
Removed